backend/config/urls.py

```diff
@@ -26,9 +26,11 @@
 
     # API endpoints
     path('api/example/<int:example_id>', views.get_example),
+    path('api/get_sentiment_analysis/<str:text>', views.get_sentiment_analysis),
```

Comment:
I believe this may raise an exception in the view function, since it only takes in the `Request` object and we'd be passing in the additional `text` string. The view function, `SentimentAnalysisAPITests`, and the React component in PR #4 interpret the `text` parameter as a URL query parameter instead of a captured value here (see [the DRF `Request.query_params`](https://www.django-rest-framework.org/api-guide/requests/#query_params), [Django's equivalent attribute](https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.GET), [the URLconf example in the Django docs](https://docs.djangoproject.com/en/3.1/topics/http/urls/#example), and [a more detailed explanation on the differences](https://stackoverflow.com/questions/150505/capturing-url-parameters-in-request-get) (the last link uses `re_path`s instead of `path`s, but the idea is the same).

I haven't tested if the above actually throws an error, but this should line up with the `APITestCase` and the view function:
```suggestion
    path('api/get_sentiment_analysis', views.get_sentiment_analysis),
```
Here, you'd be catching URLs that look more like `api/get_sentiment_analysis?text=some-text`. The [MDN docs](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL) have some very detailed articles on URLs that might help!

One last thing that came to mind is the possibility of an empty `text` parameter in the URL. I guess there are a number of ways to handle that error, but that might be left to the frontend (e.g., not accepting empty input)?

Reply:
Ah yes, we forgot to update the `urls.py` file after we made changes in `views.py`. Thank you for the catch! Frontend already ensures that `text` isn't empty so there won't be the possibility for an empty `text` parameter.